mriqc/utils/bids.py

```diff
@@ -81,6 +80,11 @@ def write_derivative_description(bids_dir, deriv_dir):
     if fname.exists():
         orig_desc = json.loads(fname.read_text())
 
+    if "Name" in orig_desc:
+        desc["Name"] = f"MRIQC - {orig_desc["Name"]}"
```

Comment:
Cant use double quotes if the f-string is using them already:

```suggestion
        desc["Name"] = f"MRIQC - {orig_desc['Name']}"
```